src/test/java/training/helpers/Random.js

```diff
@@ -0,0 +1,6 @@
+function getRandomInt(articles) {
```

Comment:
change the name of `articles` to another because you can reuse this function for users, tags, comments, whatever, and it's not necessary for articles